digits/dataset/images/classification/forms.py

```diff
@@ -21,13 +21,16 @@ class ImageClassificationDatasetForm(ImageDatasetForm):
                                   choices=[
                                       ('lmdb', 'LMDB'),
                                       ('hdf5', 'HDF5'),
+                                      ('tfrecords', 'TFRecords'),
```

Comment:
The generator actually works. But I don't want to support it. So block out the item so people can't click on it anymore.